### ISSUE TITLE
Internal error if to many text is added to advanced / custom nginx configuration

### ISSUE THREAD

**Issue by mgutt** [NginxProxyManager/nginx-proxy-manager#1227]
I've tried to add a list of IPs which are allowed to access my server. After pressing "save" it returns only "Internal Error":

![image](https://user-images.githubusercontent.com/10757176/125112124-0c198b80-e0e7-11eb-89de-68b8f080904b.png)

This does not happen if I add only a handful lines, so I think this is related to the amount / size of code I tried to add:

![image](https://user-images.githubusercontent.com/10757176/125112227-2c494a80-e0e7-11eb-9e44-563dd0b27cb8.png)

**Comment by the1ts** [comment]
Yes, this is an error, the custom nginx configuration should be a valid configuration snippet not just a list of nginx configuration. 
For simple allow ranges you should use the Access Lists feature. It allows the creation of those lists and then go to the proxy hosts you want to protect and the dropdown for access list will have the newly created access list.
